strategy-cache-and-update/README.md

```diff
@@ -0,0 +1,22 @@
+# Cache and update
+The recipe provides a service worker answering from cache to deliver fast
+responses but at the same time updating the cache entry from network.
+
+## Difficulty
+Beginner
+
+## Use Case
+You want to instantly show content and you don't mind to be temporary out of
+sync with the server.
+
+## Solution
+Serve the content from the cache but at the same time, perform a network request
```

Comment:
you don't technically do these at the same time, I'd suggest rewrite as "and also perform a network request to get fresh data to update the cache entry"